### PR TITLE
Austin.intro.2.Dukette.Barbara

### PR DESCRIPTION
**Pull request by Bjdukette**

### CODE REVIEW

File: 03LessonThree/in-class-project/daily-planet.html [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+                    white giant kitten (aka "Hello Kitty") waddled down our Main Street.</p>
+                    <p>Superman, hearing the invasion with his super hearing, spead to the scene of <thead>
+                      litter box and lured the kitty away with a lure of catnip.</p>
+
+
+            </article>
+        </section>
+        <aside><img src="https://ll-us-i5.wal.co/asr/cf754d7a-a16e-45b0-8026-b1854d847a95_1.210d9d1e3fc51f14e8d1fe5d8aea9594.jpeg?odnHeight=450&odnWidth=450&odnBg=FFFFFF" />
+      </aside>
+        <footer>This is the footer. Insert a table here with columns for the various social media links</footer>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:35]: Barbara,

Great job using all the elements required and picking a fun news story that ties into the theme of the assignment. The only thing missing from the assignment requirements was:

"Add copyright of company and current year in footer". If you update this assignment with the copyright, I can update your grade!

Other than that on small addition, good job!

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+      /><p>
+      <h2>Favorite Websites</h2>
+            <ul><li><a href="https://www.geocaching.com" >Geocaching.com</a></li>
+            <li><a href="https://google.com" >Google.com</a></li>
+          </ul>
+
+  </section>
 
 </body>
 </html>
```
Comment (ethanstenis) [inline 02LessonTwo/homework/index.html:43]: Great work, Barbara! Geocaching looks fun. Have you gotten any cool stuff from it?

File: 04LessonFour/in-class-project/css/style.css [hunk 3]
```diff
@@ -1,84 +1,22 @@
...
-#two h2 {
-    text-align: center;
 }
-
-#two h3 {
-    text-decoration: underline;
+.support {
+    background-color: grey;
+    margin-top: -6x;
+    Padding-left: 10px;
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:15]: "Padding-left" is not being recognized. Simply change the "P" to a "p" - "padding-left" should work here.

File: 04LessonFour/in-class-project/css/style.css [hunk 4]
```diff
@@ -1,84 +1,22 @@
...
-    font-family: sans-serif;
-    font-size: 15px;
+    margin: auto;
+    width: 80%;
+    font-family: Helvetica, Arial, sans-serif;
+    margin-bottom: 10px;
+}
+#header {
+    font-size: 24px;
+    background-color: #0071c5;
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:9]: Add "padding-left: 10px;" to #header

File: 07LessonSeven/homework/css/style.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
+body {
+    font-family: sans-serif;
```
Comment (ethanstenis) [inline 07LessonSeven/homework/css/style.css:2]: You are just a few CSS changes away from completing this assignment with 100% grade. Let me know if you need any assistance.

File: 11LessonEleven/homework/inspired.html [hunk 6]
```diff
@@ -3,12 +3,31 @@
...
+        <article class="mumbai">
+            <h1>Everyday Food For Mumbai Children</h1>
+            <p>Macaroon chocolate bar croissant tiramisu jelly sweet topping gummies cookie. Macaroon halvah sugar plum tart gummi bears fruitcake sugar plum. Pastry biscuit fruitcake wafer carrot cake jujubes cake.</p>
+        </article>
+    </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (ethanstenis) [inline 11LessonEleven/homework/inspired.html:33]: Great job! Just need to change some font sizes in your phone media queries.
screen shot 2016-09-13 at 6 04 57 pm

File: 12LessonTwelve/homework/index.html [hunk 7]
```diff
@@ -1,10 +1,66 @@
...
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" /></script>
 </body>
+
```
Comment (ethanstenis) [inline 12LessonTwelve/homework/index.html:66]: Looks like that pricing button is going crazy! Let's get that settled into place and I can bump your grade up. The Dev Tools is a great way to troubleshoot this issue.
screen shot 2016-09-13 at 6 05 59 pm

File: 11LessonEleven/in-class-project/index.html [hunk 8]
```diff
@@ -15,28 +15,28 @@
...
         </div>
         <div class="box">
             <div class="aqua"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
             </div>
         </div>
     </main>
```
Comment (ethanstenis) [inline 11LessonEleven/in-class-project/index.html:42]: Just need some CSS on that H1 in the phone media query. Let's get that fixed and I can bump your grade up.
screen shot 2016-09-13 at 6 05 24 pm

File: 12LessonTwelve/homework/index.html (hunk 7 above)
Comment (Bjdukette) [reply 12LessonTwelve/homework/index.html:66]: Hi Luke, I don't know why this looks ok on my screen!

---

On Tue, 9/13/16, Ethan Stenis notifications@github.com wrote:

 Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.intro.2.Dukette.Barbara (#92)
 To: "AustinCodingAcademy/intro-workbook" intro-workbook@noreply.github.com
 Cc: "Bjdukette" Bjdukette@yahoo.com, "Author" author@noreply.github.com
 Date: Tuesday, September 13, 2016, 6:07 PM

 In 12LessonTwelve/homework/index.html:

>  </body>
> +

 Looks like that pricing button is going crazy! Let's
 get that settled into place and I can bump your grade up.
 The Dev Tools is a great way to troubleshoot this issue.

 —
 You are receiving this because you authored the thread.
 Reply to this email directly, view
 it on GitHub, or mute
 the thread.

File: 11LessonEleven/in-class-project/index.html (hunk 8 above)
Comment (Bjdukette) [reply 11LessonEleven/in-class-project/index.html:42]: Hi Luke,

Just checking my grades -

Lesson 7 in class 0?
Lesson 9 in class 50?

I'm missing something ... I did both projects and turned them in

Still waiting to hear back from github ...

## Barbara

On Tue, 9/13/16, Ethan Stenis notifications@github.com wrote:

 Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.intro.2.Dukette.Barbara (#92)
 To: "AustinCodingAcademy/intro-workbook" intro-workbook@noreply.github.com
 Cc: "Bjdukette" Bjdukette@yahoo.com, "Author" author@noreply.github.com
 Date: Tuesday, September 13, 2016, 6:08 PM

 In 11LessonEleven/in-class-project/index.html:

> ```
>        </div>
>      </div>
>      <div class="box">
>          <div
> ```
> 
>  class="aqua"></div>
>              <div class="container">
>                  <h1>Title</h1>
> -                <p>Pudding sweet roll jelly-o
>   chupa chups cheesecake lollipop muffin cake. Cupcake muffin
>   chocolate bar soufflé donut sesame snaps toffee jujubes.
>   Biscuit danish jelly-o jelly-o cupcake candy powder
>   chocolate pie. </p>
> -                <p>Pudding sweet roll jelly-o
>   chupa chups cheesecake lollipop muffin cake. Cupcake muffin
>   chocolate bar soufflé donut sesame snaps toffee jujubes.
>   </p>
>              </div>
>          </div>
>      </main>

 Just need some CSS on that H1 in the phone media query.
 Let's get that fixed and I can bump your grade up.

 —
 You are receiving this because you authored the thread.
 Reply to this email directly, view
 it on GitHub, or mute
 the thread.
